src/libcollections/vec.rs

```diff
@@ -456,8 +458,9 @@ impl<T> Vec<T> {
     }
 
     /// Reserves the minimum capacity for exactly `additional` more elements to
-    /// be inserted in the given `Vec<T>`. Does nothing if the capacity is already
-    /// sufficient.
+    /// be inserted in the given `Vec<T>`. After calling `reserve_exact`,
+    /// capacity will be greater than or equal to `self.len() + additional`.
```

Comment:
> otherwise ensures the capacity is equal to self.len() + additional upon returning

I'm pretty sure we don't guarantee that at the moment. Do we want to start guaranteeing that now? Other collections have a `reserve_exact` method which acts the same so any changes should be made to all of them, not just `Vec`.